strings-csharp/HTMLContentString/HTMLContentString/HtmlHttp.cs

```diff
@@ -0,0 +1,21 @@
+namespace HTMLContentString;
+
+public class HtmlHttp
+{
+    private readonly HttpClient _httpClient;
+
+    public HtmlHttp()
+    {
+        _httpClient = new HttpClient();
```

Comment:
@emekaemego 
We are instantiating HttpClient by using the new keyword, which is far from best practice. We should be using HttpClientFactory injected into the class to retrieve an instance of HttpClient. 

We only need to do a couple of things to our project:
Add a NuGet reference to Microsoft.Extensions.Hosting + Microsoft.Extensions.Http: https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection-usage#create-a-new-console-application

Update the Program class to use a HostApplicationBuilder and register the HttpClient: https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection-usage#register-services-for-di